stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Thank you for this great story Jack, it reads supper well.</p><p> </p><p>I think the conclusion is missing a discussion (just a paragraph after the first one) about the limitations of these forecast (e.g model limitations, assumptions that might not be completely true, or that might change in time, etc). You did this a bit at the end of the "Alternative forecast" section but i think it needs a general overview.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
<p>Agreed - great story, really interesting analysis, clear explanation, super timely.</p><p> </p><p>Maybe part of this discussion is exploring what contributes to the vaccination rate: supply (doses acquired by the UK gov from the vaccine makers), demand (people getting the vaccine when offered), and process (roll-out, vaccination centres, appointments). I'm still mostly concerned about statements that imply vaccination has been completed with certain groups when there might still be some hesitancy/resistance in these groups.</p>

Reply:
<p>Also, I just made some copyedits through Github - hope that was the right approach. Not sure what to do next. https://github.com/alan-turing-institute/TuringDataStories/compare/covid-vax-forecast...billfinnegan:patch-1</p>

Reply:
Thanks @billfinnegan ! I'll merge in those edits and then start working through the rest of your comments 🙂